Source/JavaScriptCore/assembler/X86Assembler.h

```diff
@@ -5008,6 +5041,16 @@ class X86Assembler {
         m_formatter.immediate8(static_cast<uint8_t>(rounding));
     }
 
+    void vsqrtps_rr(XMMRegisterID src, XMMRegisterID dest)
+    {
```

Comment:
Let's add the comment aligned to the other AVX functions, it is,
// URL
// Encoding
// Operand interpretation